foundry.toml

```diff
@@ -90,12 +60,12 @@
 
 [rpc_endpoints]
   # mainnets
-  arbitrum = "https://rpc.ankr.com/arbitrum"
-  avalanche = "https://rpc.ankr.com/avalanche"
+  arbitrum = "https://arbitrum-mainnet.infura.io/v3/${API_KEY_INFURA}"
+  avalanche = "https://avalanche-mainnet.infura.io/v3/${API_KEY_INFURA}"
   base = "https://mainnet.base.org"
   berachain = "https://rpc.berachain.com"
   blast = "https://rpc.blast.io"
-  bnb = "https://rpc.ankr.com/bsc"
+  bnb = "https://bsc-mainnet.infura.io/v3/${API_KEY_INFURA}"
```

Comment:
Can we rename it to `bsc` since thats the terminology we are using across docs and sdk?